server/services/interventionsService.test.ts

```diff
@@ -72,6 +72,36 @@ pactWith({ consumer: 'Interventions UI', provider: 'Interventions Service' }, pr
       })
     })
 
+    describe('for a referral that has had a service provider selected', () => {
+      beforeEach(async () => {
+        await provider.addInteraction({
+          state:
+            'There is an existing draft referral with ID of d496e4a7-7cc1-44ea-ba67-c295084f1962, and it has had a service provider selected',
+          uponReceiving: 'a request for that referral',
+          withRequest: {
+            method: 'GET',
+            path: '/draft-referral/d496e4a7-7cc1-44ea-ba67-c295084f1962',
+            headers: { Accept: 'application/json', Authorization: `Bearer ${token}` },
+          },
+          willRespondWith: {
+            status: 200,
+            body: Matchers.like({
+              id: 'd496e4a7-7cc1-44ea-ba67-c295084f1962',
+              serviceProviderId: '674b47a0-39bf-4514-82ae-61885b9c0cb4',
```

Comment:
_thinking out loud, not a blocker_
This could be `selectedServiceProviderId` to reinforce that we elect one of them to be the "recipient" of the referral. The context of provider/referral might be obvious as it is, though.

Reply:
I think that if there were multiple different ways in which a service provider could be related to a referral, then it would make sense to disambiguate them, but I'm struggling to think of what these might be.